datastore/longhorn.go

```diff
@@ -3790,3 +3790,28 @@ func (s *DataStore) ListOrphansByNodeRO(name string) ([]*longhorn.Orphan, error)
 func (s *DataStore) DeleteOrphan(orphanName string) error {
 	return s.lhClient.LonghornV1beta2().Orphans(s.namespace).Delete(context.TODO(), orphanName, metav1.DeleteOptions{})
 }
+
+func (s *DataStore) UpdatePVAnnotation(volume *longhorn.Volume, annotationKey, annotationVal string) error {
```

Comment:
This is for updating a builtin resource, so will it be better to move to datastore/kubernetes.go?

cc @c3y1huang

Reply:
Updated, thank you.